pyproject.toml

```diff
@@ -16,6 +16,21 @@ ignore_missing_imports = true
 plugins = ["mypy_django_plugin.main", "mypy_drf_plugin.main"]
 python_version = "3.10"
 strict = true
+disallow_untyped_calls = false
+warn_return_any = false
+
+[[tool.mypy.overrides]]
+module = [
+  "tests.*"
+]
+ignore_errors = true
+
+[mypy.plugins.django-stubs]
```

Comment:
```suggestion
[tool.mypy.plugins.django-stubs]
```
or not?

Reply:
I think if it works - we shouldn't touch it :D

Reply:
I've tested and your solution breaks it :)